scipy/stats/_distr_params.py

```diff
@@ -37,6 +37,7 @@
     ['gengamma', (4.4162385429431925, 3.1193091679242761)],
     ['gengamma', (4.4162385429431925, -3.1193091679242761)],
     ['genhalflogistic', (0.77274727809929322,)],
+	['genhyperbolic', (1.0,1.0,0.0,1.0,0.0,)],
```

Comment:
```suggestion
    ['genhalflogistic', (0.77274727809929322,)],
    ['genhyperbolic', (1.0, 1.0, 0.0, 1.0, 0.0,)],
```
Need to use spaces instead of tabs.
Use [`flake8`](https://flake8.pycqa.org/en/latest/) to find [PEP8](https://www.python.org/dev/peps/pep-0008/) issues in the parts you added or modified. You don't need to correct everything in the file (there are probably many other errors), but new code needs to be properly formatted.

Reply:
You can see results of PEP8 checks by clicking the details link of **scipy.scipy (Lint)** :
![image](https://user-images.githubusercontent.com/6570539/103185047-79627f00-486f-11eb-984b-89eebadca255.png)

But I would recommend using `flake8` before pushing.

I think most of these are from this new code:
```
scipy/integrate/_quad_vec.py:271:5: E303 too many blank lines (2)
scipy/stats/_continuous_distns.py:2966:5: E115 expected an indented block (comment)
scipy/stats/_continuous_distns.py:2977:5: E115 expected an indented block (comment)
scipy/stats/_continuous_distns.py:3000:5: E303 too many blank lines (2)
scipy/stats/_continuous_distns.py:3001:5: E115 expected an indented block (comment)
scipy/stats/_distr_params.py:40:1: E101 indentation contains mixed spaces and tabs
scipy/stats/_distr_params.py:40:1: W191 indentation contains tabs
scipy/stats/_distr_params.py:40:2: E131 continuation line unaligned for hanging indent
scipy/stats/_distr_params.py:41:1: E101 indentation contains mixed spaces and tabs
scipy/stats/stats.py:4598:5: E303 too many blank lines (2)
scipy/stats/tests/test_distributions.py:382:9: E115 expected an indented block (comment)
scipy/stats/tests/test_distributions.py:383:9: E115 expected an indented block (comment)
scipy/stats/tests/test_distributions.py:394:9: E115 expected an indented block (comment)
scipy/stats/tests/test_distributions.py:395:9: E115 expected an indented block (comment)
scipy/stats/tests/test_distributions.py:409:66: E502 the backslash is redundant between brackets
scipy/stats/tests/test_distributions.py:410:97: E502 the backslash is redundant between brackets
scipy/stats/tests/test_distributions.py:411:67: E502 the backslash is redundant between brackets
scipy/stats/tests/test_mstats_basic.py:353:5: E303 too many blank lines (2)
scipy/stats/tests/test_mstats_basic.py:366:5: E303 too many blank lines (2)
scipy/stats/tests/test_mstats_basic.py:382:5: E303 too many blank lines (2)
scipy/stats/tests/test_mstats_basic.py:398:5: E303 too many blank lines (2)
scipy/stats/tests/test_stats.py:3525:5: E303 too many blank lines (2)
scipy/ndimage/interpolation.py:801:17: E124 closing bracket does not match visual indentation
scipy/ndimage/tests/test_interpolation.py:855:5: E303 too many blank lines (2)
scipy/ndimage/tests/test_interpolation.py:870:5: E303 too many blank lines (2)
2       E101 indentation contains mixed spaces and tabs
7       E115 expected an indented block (comment)
1       E124 closing bracket does not match visual indentation
```